CHANGELOG.md

```diff
@@ -1,5 +1,70 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.14.11
+
+<!-- generated comparing v5.14.10..master -->
+_Sep 25, 2023_
+
+A big thanks to the 17 contributors who made this release possible.
+This release was mostly about 🐛 bug fixes and 📚 documentation improvements.
+
+### `@mui/material@5.14.11`
+
+- [Autocomplete] Re-export `AutocompleteValue` to make it available from path import (#38638) @vadimka123
+- [Select] Missing aria-multiselectable attribute on multiple Select component (#38855) @gitstart
+- [Skeleton] Add new props to the Skeleton component (#38483) @gitstart
+
+### `@mui/base@5.0.0-beta.17`
+
+- [NumberInput] Support adornments (#38900) @anle9650
+- [Menu] Align external props handling for useMenu/MenuButton/MenuItem (#38946) @mj12albert
+- [Select] Align external props handling (#39038) @mj12albert
+- [TextareaAutosize] Simplify logic and add test (#38728) @oliviertassinari
+
+### `@mui/material-next@6.0.0-alpha.103`
+
+- [FormControl] Copy v5 FormControl (#39039) @mj12albert
+
+### `@mui/lab@5.0.0-alpha.146`
+
+- [TreeView] Fix JSDoc comments in TreeView and TreeItem  (#38874) @jergason
+
+### Docs
```

Comment:
Sort of similar to what we're doing with the Core section, should we discriminate for which product these documentation-related improvements are for? Some of them are general doc changes (e.g., "Improve the default theme viewer design"), while others are Joy UI-focused (e.g., "Refine the Sign in template", "Revise the Drawer page", etc.), and so on.

Reply:
@DiegoAndai is working on a proposal for the rules that should apply on the changelog. Until we decide on it, we can keep it as is I think.

Reply:
I don't know. Would you find it useful? TBH I don't think including doc-related PRs in the changelog brings value to developers. In my opinion, it would be sufficient to include only the changes in packages.

Reply:
I think it's at least sweet to signal that there are constant and frequent documentation improvements; it's a crucial part of the developer experience. But, cool, I'm down to hold changes until the proposal is sorted. 🤙